openpype/hosts/maya/plugins/create/create_animation_pointcache.py

```diff
@@ -0,0 +1,225 @@
+        )
+    )
+
+    abc_defs.append(
+        EnumDef(
+            "dataFormat",
+            label="Data Format",
+            default=getattr(cls, "dataFormat", None),
+            items=["ogawa", "HDF"],
+            disabled=True if "dataFormat" not in abc_args_overrides else False,
+        )
+    )
+
+    abc_defs.append(
+        NumberDef(
+            "preRollStartFrame",
+            label="Start frame for preroll",
+            default=getattr(cls, "preRollStartFrame", None),
+            tooltip=(
+                "The frame to start scene evaluation at. This is used to set"
+                " the starting frame for time dependent translations and can"
+                " be used to evaluate run-up that isn't actually translated."
+            ),
+            disabled=True
+            if "preRollStartFrame" not in abc_args_overrides
+            else False,
+        )
+    )
+
+    return abc_defs
+
+
+class CreateAnimation(plugin.MayaHiddenCreator):
+    """Animation output for character rigs
+
+    We hide the animation creator from the UI since the creation of it is
+    automated upon loading a rig. There's an inventory action to recreate it
+    for loaded rigs if by chance someone deleted the animation instance.
+    """
+
+    identifier = "io.openpype.creators.maya.animation"
+    name = "animationDefault"
+    label = "Animation"
+    family = "animation"
+    icon = "male"
+
+    write_color_sets = False
+    write_face_sets = False
+    include_parent_hierarchy = False
+    include_user_defined_attributes = False
+
+    def collect_instances(self):
+        pass
+
+    def get_instance_attr_defs(self):
+        super(CreateAnimation, self).get_instance_attr_defs()
+        # adding project settings, since MayaHiddenCreator does not
+        # handle this for us (yet?)
+        settings = (
+            getattr(self, "project_settings", {})
+            .get("maya", {})
+            .get("create", {})
+            .get("CreateAnimation")
+        )
+
+        for key, value in settings.items():
+            setattr(self, key, value)
+
+        defs = _get_animation_attr_defs(self)
+
+        abc_defs = _get_animation_abc_attr_defs(self)
+
+        if abc_defs:
+            defs.extend(abc_defs)
+
+        return defs
+
+
+class CreatePointCache(plugin.MayaCreator):
+    """Alembic pointcache for animated data"""
+
+    identifier = "io.openpype.creators.maya.pointcache"
+    label = "Pointcache"
+    family = "pointcache"
+    icon = "gears"
+    write_color_sets = False
+    write_face_sets = False
+    include_user_defined_attributes = False
+
+    def get_instance_attr_defs(self):
+        super(CreatePointCache, self).get_instance_attr_defs()
+        # defs = self.get_instance_attr_defs()
+        print(self.instance_attr_defs)
+        defs = _get_animation_attr_defs(self)
+
+        abc_defs = _get_animation_abc_attr_defs(self)
+
+        if abc_defs:
+            defs.extend(abc_defs)
+
+        return defs
```

Comment:
```suggestion
        defs = super(CreatePointCache, self).get_instance_attr_defs()
        defs.extend(_get_animation_attr_defs(self))
        abc_defs = _get_animation_abc_attr_defs(self)
        if abc_defs:
            defs.extend(abc_defs)

        return defs
```

Reply:
is line 288 you suggested duplicated with what you suggested in 289-291?

Reply:
Oof, this has been a while. Not sure what you're asking exactly. I'd have to re-read the PR's code to understand whether your suggestion is better or not.